tests/test_dataframe.py

```diff
@@ -482,3 +482,59 @@ def tests_merge_left_compound_key(self):
             self.assertEqual(expected, ddf['r_vals'].data[:])
             self.assertEqual(ddf['l_id_1'].data[:].tolist(), ddf['r_id_1'].data[:].tolist())
             self.assertEqual(ddf['r_id_2'].data[:].tolist(), ddf['r_id_2'].data[:].tolist())
```

Comment:
I know you didn't change this but I noticed a bug in the test here. Given that I don't want to introduce a merge conflict and I imagine this would do so, can you change line 484 so it is 'l_id_2' on the left hand side?

Reply:
fixed this bug